frontend/catalyst/pennylane_extensions.py

```diff
@@ -227,6 +227,8 @@ class QJITDevice(qml.QubitDevice):
         "Toffoli",
         "MultiRZ",
         "QubitUnitary",
+        "ISWAP",
+        "PSWAP",
```

Comment:
Let's ensure these gates are now supported from the frontend. You can check the support for lightning simulators in `test_operations.py` and for openqasm devices in `test_braket_local_devices.py` and `test_braket_aws_devices.py`.

Reply:
I would also add that we should write a test that verifies the gates are not decomposed :)

Reply:
Thanks for adding some tests! The decomposition aspect happens in the frontend actually, so it would be great to have a test there that checks the gates are not decomposed (rather than in the runtime). 

In this case, you could for example check that `ISWAP` and `PSWAP` appear in the jaxpr of a test case. 

Tests that verify something about our generated IR are well suited for lit/FileCheck tests. Have a look at our existing lit tests to see how to write one: https://github.com/PennyLaneAI/catalyst/tree/main/frontend/test/lit